_posts/2019-02-25-the-western-grid.md

```diff
@@ -0,0 +1,136 @@
+
+<div class="caption">
+  <a href="{{ "/images/principal_meridians.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/principal_meridians_sm.png" | prepend: site.baseurl }}" alt="PLSS Principal Meridians">
+  </a>
+  <p class="caption-text">Fig 4: PLSS Principal Meridians</p>
+</div>
+<div class="caption">
+  <img src="{{ "/images/salt_lake_meridian.png" | prepend: site.baseurl }}" alt="Salt Lake Meridian" />
+  <p class="caption-text">Fig 5: Salt Lake Meridian</p>
+</div>
+
+As with any measuring system, you need a starting point. Within the PLSS, this is called the Principal Meridian. In Utah, this location is in Salt Lake City at Main Street and South Temple (Fig 5). All official surveys in Utah have an underlying reference to this point. In other words, every Township and Section in Utah is either North, South, East, or West of this location. Other PLSS states have their own unique principal meridians (Fig 4).
+
+The Townships are made up of six square miles, with each square mile known as a Section. These Sections are eventually subdivided down to an acre by way of Half Sections, Quarter Sections, Quarter-Quarter Sections, etc. (Fig 6). An acre, again, is simply 10 square chains - and there are 640 acres in a Section (a square-mile) or 40 acres in a quarter-quarter section. Hence the phrase, “40 acres and mule” is all you needed to be self-sufficient.
+
+<div class="caption">
+  <img src="{{ "/images/townships_sections.png" | prepend: site.baseurl }}" alt="Townships and Sections" />
+  <p class="caption-text">Fig 6: Townships, Ranges and Sections</p>
+</div>
+
+No matter how you shake it, though, land measurements in the West are consistently divisible by the magic 66 foot Gunter Chain. You’ll see its effect everywhere. This measuring unit was so successful because it made the math easy - and many early surveyors had a shaky grasp of mathematics and they were required to work quickly.
+
+These early surveyors typically placed a permanent monument at Section and Quarter-Section corners. These markers are still heavily used today and are the starting point for almost every Western legal description you’ll encounter - including land deeds (the transfer of private property), water rights (well locations), taxing areas, municipal annexations - and consequently voting districts. Because of this, it has been difficult for the US to adopt the International System of Units (the modern metric system).
```

Comment:
```suggestion
These early surveyors typically placed a permanent monument at section and quarter-section corners. These markers are still heavily used today and are the starting point for almost every Western legal description you’ll encounter, including land deeds (the transfer of private property), water rights (well locations), and taxing areas, as well as municipal annexations and, consequently, voting districts. Because of this, it has been difficult for the US to adopt the International System of Units (i.e., the modern metric system).
```